.pre-commit-config.yaml

```
@@ -13,7 +13,7 @@ repos:
         args:
           - --py39-plus
   - repo: https://github.com/psf/black
```

```suggestion
  - repo: https://github.com/psf/black-pre-commit-mirror
```
https://black.readthedocs.io/en/stable/integrations/source_version_control.html

---

Thanks!